megamek/src/megamek/server/Server.java

```diff
@@ -12125,6 +11660,7 @@ private boolean checkVibrabombs(Entity entity, Coords coords,
      * @param coords
      *            The <code>Coords</code> from which to remove minefields
      */
+    // TODO : Windchild Can this be removed?
```

Comment:
Can this commented out method be removed?

Reply:
That's been commented out for over ten years. Dump it.